js/views/itemEditVw.js

```diff
@@ -314,11 +333,21 @@ module.exports = Backbone.View.extend({
     } else {
       uploadMsg.removeClass('hide');
     }
+
+    if (imageArray.length >= this.MAX_PHOTOS) {
+      $('.js-itemImageUpload').prop('disabled', true)
```

Comment:
There's more required here. 

If you upload 10 photos, then delete one of them, the button reactivates. Then you can upload an arbitrary number of extra photos (in my case 4 more, for a total of 13). 

There will have to be a check in resizeImage to not add images over 10. 

It would also be good to show an error message when someone tries to upload more than 10 images, so they know why it doesn't work.